src/pages/HomePage/HomePage.style.ts

```diff
@@ -0,0 +1,27 @@
+import styled from 'styled-components';
+
+const HomePageContainer = styled.div`
+  display: flex;
+  justify-content: center;
+`;
+
+const HomePageWrapper = styled.div`
+  width: 128rem;
+`;
```

Comment:
위 속성이 바뀌어서 이부분도 이렇게 수정해주는게 필요할 것 같습니다!

최대 너비를 지정해주고, margin: auto를 통해 최대 너비 이상의 해상도에서는 가운데 정렬을 해주는 코드입니다!

```suggestion
const HomePageWrapper = styled.div`
  display: flex;
  flex-direction: column;
  align-items: center;
  max-width: 128rem;
  margin: auto;
`;
```

Reply:
margin: auto; 설정을 생각 못해서 조금 하드코딩 했던 것 같은데, 덕분에 쉽게 처리할 수 있는 방법을 알게되었어요 😄 

또한 width를 max width로 수정하는 방법도 좋아보입니다 👍